ArduPlane/tailsitter.cpp

```diff
@@ -114,13 +114,21 @@ void QuadPlane::tailsitter_output(void)
     // handle VTOL modes
     // the MultiCopter rate controller has already been run in an earlier call
     // to motors_output() from quadplane.update(), unless we are in assisted flight
-    if (assisted_flight) {
+    if (assisted_flight && tailsitter_transition_fw_complete()) {
         hold_stabilize(SRV_Channels::get_output_scaled(SRV_Channel::k_throttle) * 0.01f);
         motors_output(true);
+
+        if ((tailsitter.options & TAILSITTER_Q_ASSIST_MOTORS_ONLY) != 0) {
+            // only use motors for Q assist, control surfaces remain under plane control
+            // zero copter I terms and use plane
+            attitude_control->reset_rate_controller_I_terms();
+            return;
```

Comment:
took me a while to notice the early return here. At first I wondered how clearing the integrator was all you needed to do :)

Reply:
I guess it makes sense that assisted_flight bypasses the manual control stuff down below.

Reply:
doesn't this kill all of the vectored hover code?

Reply:
yes, it does, I have added back in tilt vectoring